lib/ex_unit/lib/ex_unit/case.ex

```diff
@@ -438,28 +451,41 @@ defmodule ExUnit.Case do
     {name, describe, describe_line, describetag} =
       case Module.get_attribute(mod, :ex_unit_describe) do
         {line, describe} ->
-          {:"#{type} #{describe} #{name}", describe, line, Module.get_attribute(mod, :describetag)}
+          {
+            :"#{type} #{describe} #{name}",
```

Comment:
Please move the first element to a variables as well so everything fits a single line,

Reply:
```
    description = :"#{type} #{describe} #{name}"

    {name, describe, describe_line, describetag} =
      case Module.get_attribute(mod, :ex_unit_describe) do
        {line, describe} ->
          {description, describe, line, Module.get_attribute(mod, :describetag)}
        _ ->
          {:"#{type} #{name}", nil, nil, []}
      end
```
@josevalim something like that?

Reply:
This would be better:

```
    {name, describe, describe_line, describetag} =
      case Module.get_attribute(mod, :ex_unit_describe) do
        {line, describe} ->
          description = :"#{type} #{describe} #{name}"
          {description, describe, line, Module.get_attribute(mod, :describetag)}

        _ ->
          {:"#{type} #{name}", nil, nil, []}
      end
```

Please don't forget to run the formatter after your changes. :)

Reply:
ok